Vostok.Hosting/Components/Log/LogEventsMetrics.cs

```diff
@@ -0,0 +1,28 @@
+﻿using JetBrains.Annotations;
+
+namespace Vostok.Hosting.Components.Log
+{
+    [PublicAPI]
+    public class LogEventsMetrics
```

Comment:
An optional suggestion: wouldn't it be simpler if we just had a `Dictionary<LogLevel, int>`? Looks like `EventLevelCounter` would be much more concise if it didn't have to account for each level personally. And `LogLevelMetrics` would probably benefit from removal of reflection.

Reply:
I see the point, but this approach has its cons:

1) The user will need to check the presence of values in LogEventsMetrics's dictionary (because they might not be initialized). (Or we will have to initialize this values for the user so either reflection or per level check returns there).

2) EventLevelCounter will be a little bit slower, because it requires to check the presence of a counter in the dictionary, create one if it is absent, and then increment the value.

3) To collect metrics, we have to access the same dictionary, so there might be problems during enumeration (if someone adds a new value to the dictionary).

 And for me, personally, it just feels more robust to see concrete values instead of dictionary in the model :)